stubs/CMakeLists.txt

```diff
@@ -14,6 +14,16 @@ ENDIF()
 
 PROJECT(@lib_name@)
 
+# check for and enable c++11 support
+INCLUDE(CheckCXXCompilerFlag)
+CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+IF(COMPILER_SUPPORTS_CXX11)
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
@scopatz I am willing to PR this changeset into `conda-forge/cyclus-feedstock`. But I am not sure to understand how :s.

Secondly, What is was trying to highlight with my previous quote is that` OS X use by default libc++ and the  -stdlib=libc++ is not required`. (There is maybe something I did not get from you reply after)